ci-operator/populate-secrets-from-bitwarden.sh

```diff
@@ -137,15 +137,3 @@ for registry in "docker.io" "quay.io" "quay.io/openshift-knative" "quay.io/opens
 	# we want to be able to build and push out to registries
 	oc secrets link builder "registry-push-credentials-${registry//\//\-}"
 done
-
-target_cloud="azure"
```

Comment:
@stevekuznetsov 

Based on our email thread with Jim, I feel we can remove this since `azure` (and `azure4`) is out of sync anyway.